Dockerfile

```diff
@@ -39,21 +42,17 @@ ENV ZB_HOME=/usr/local/zeebe \
     ZEEBE_STANDALONE_GATEWAY=false
 ENV PATH "${ZB_HOME}/bin:${PATH}"
 
-RUN groupadd -g 1000 zeebe && \
-    adduser -u 1000 zeebe --system --ingroup zeebe
-
 WORKDIR ${ZB_HOME}
 EXPOSE 26500 26501 26502
 VOLUME ${ZB_HOME}/data
 
-COPY --chown=1000:0 docker/utils/startup.sh /usr/local/bin
-RUN chgrp 0 /usr/local/bin/startup.sh && chmod g=u /etc/passwd && chmod 0775 /usr/local/bin/startup.sh
-COPY --from=builder --chown=1000:0 /tmp/zeebe ${ZB_HOME}
-RUN chown 1000:0 -R ${ZB_HOME}/ && chmod 0775 ${ZB_HOME}/ ${ZB_HOME}/data
+RUN groupadd -g 1000 zeebe && \
+    adduser -u 1000 zeebe --system --ingroup zeebe
+RUN chmod g=u /etc/passwd
+RUN chown -R 1000:1000 ${ZB_HOME}
 
-# Set execution flag (otherwise, if image is built in a Windows environment, the start scripts
-# won't be executable)
-RUN chmod +x ${ZB_HOME}/bin/broker
-RUN chmod +x ${ZB_HOME}/bin/gateway
+USER zeebe
+COPY --from=builder --chown=1000:1000 /tmp/zeebe/bin/startup.sh /usr/local/bin
+COPY --from=builder --chown=1000:1000 /tmp/zeebe ${ZB_HOME}
```

Comment:
again group has to be `0`, and maybe we can just keep the `startup.sh` in the `${ZB_HOME}/bin` dir instead of copying it around as we also do it `tini`. we can adjust the `ENTRYPOINT` instead.

Reply:
Can you explain why the group has to be 0? How did you test this? The image runs fine without the 0 group, should I be testing something else?

Reply:
The group `0` or root group is recommended by the OpenShift documentation, see section https://access.redhat.com/documentation/en-us/openshift_container_platform/4.4/html/images/creating-images#images-create-guide-openshift_create-images paragraph "Support arbitrary user ids" 

> For an image to support running as an arbitrary user, directories and files that may be written to by processes in the image should be owned by the root group and be read/writable by that group. Files to be executed should also have group execute permissions.

> Adding the following to your Dockerfile sets the directory and file permissions to allow users in the root group to access them in the built image:
```
RUN chgrp -R 0 /some/directory && \
    chmod -R g=u /some/directory
```
>Because the container user is always a member of the root group, the container user can read and write these files.